### ISSUE TITLE
Deprecate datasets `cnb_classification` and `isic_classification`

### ISSUE THREAD

**Issue by jdhoffa** [RMI-PACTA/r2dii.data#329]
Relates to #327 

FYI @jacobvjk, I plan to deprecate these datasets properly, which means including a release where we just warn that they will be deprecated.

**Comment by AlexAxthelm** [comment]
@jdhoffa do we have a rough timeframe (on a calendar) when we expect to fully deprecate?

**Comment by jdhoffa** [comment]
Nope, but I think if we do my quarterly release schedule, then next release is end of Q1 (end of March). That will mark beginning of deprecation. 

Following release is end of Q2 (so end of June?)

I guess end of June would be when they are hard deprecated? I will update the warning message and add an ADO ticket

**Comment by AlexAxthelm** [comment]
Maybe have something along the lines of

"`foo` will be available until 2024-06-01. In releases of `r2dii.data` after that date, it will be unavailable. Please use `bar` instead. See <url> this blog post for more information."

So users know have some idea of when they would need to transition by?

**Comment by jdhoffa** [comment]
Deprecation warning is already there. Following the `lifecycle` style deprecation process:
https://github.com/RMI-PACTA/r2dii.data/pull/331/files

I just need to update the "date"

**Comment by jdhoffa** [comment]
@AlexAxthelm do you think "expected in 2024Q2 release" would be too vague actually? I would tend to prefer that...

**Comment by jdhoffa** [comment]
ADO ticket in proposal of next quarter's KRs involving completing the deprecation process:
https://dev.azure.com/RMI-PACTA/2DegreesInvesting/_workitems/edit/9711

**Comment by AlexAxthelm** [comment]
> "expected in 2024Q2 release" would be too vague actually?

I think I shouldn't write user-facing messages. But that's reasonable enough from my side.

cc @hodie

**Comment by jdhoffa** [comment]
As per a discussion with @jacobvjk, we are going to add the ISIC classification bridge revision 5. So rather than a "deprecation" the dataset will just be updated. 

`cnb_classification` will still be deprecated
